libraries/AP_AHRS/AP_AHRS_DCM.cpp

```diff
@@ -1036,10 +1040,13 @@ bool AP_AHRS_DCM::get_position(struct Location &loc) const
     loc.terrain_alt = 0;
```

Comment:
Pre-existing, but this method (and the one other like it) is missing `WARN_IF_UNUSED` - and some places aren't using the value that aren't and don't have a comment explaining why.  `get_secondary_position` for example.

Reply:
pre-existing, I'll let you handle that in your AHRS change sets